app/views/works/index.html.erb

```diff
@@ -0,0 +1,79 @@
+<main>
+	<h2>List of works</h2>
+	<section class="media-table">
+		<h4>Albums</h4>
+		<table class="table">
+			<thead>
+				<tr>
+					<th width="80">Votes</th>
+					<th>Title</th>
+					<th>Created By</th>
+					<th width="100">Published</th>
+					<th width="100">Upvote</th>
+				</tr>
+			</thead>
+			<tbody>
+				<% Work.albums.each do |album| %>
```

Comment:
Just noting that these types of sections could be handled with a partial view which is repeated for each category.